js/keyboard.js

```diff
@@ -0,0 +1,154 @@
+import arr from './data.js';
+import Language from './language.js';
+
+export default class Keyboard {
+	static pressCaps = false;
+
+	static pressShift = false;
+
+	static drawKeyboard() {
+		for (let i = 0; i < arr.length; i++) {
+			for (let j = 0; j < arr[i].length; j++) {
+				document.querySelector(`#keyboard_row-${i}`).innerHTML += `<button id='${arr[i][j].id}' >${arr[i][j].value[Language.checkLanguage()]}</button>`;
+			}
+		}
+	}
+
+	static rewriteKey() {
+		for (let i = 0; i < arr.length; i++) {
+			for (let j = 0; j < arr[i].length; j++) {
+				document.querySelector(`#${arr[i][j].id}`).innerText = `${arr[i][j].value[Language.checkLanguage()]}`;
+			}
+		}
+	}
```

Comment:
rewrite it and don't use static where it's not worth it
think about all the code, not only this part

Reply:
static was returned because of bugs and no time to solve them